lib/dom_model.dart

```diff
@@ -347,9 +333,7 @@ class _Node implements Node {
 List<String> flattenAttributesMap(Map<String, String> attributes) {
   var result = <String>[];
   attributes.forEach((k, v) {
-    if (k != null) {
-      result..add(k)..add(v);
-    }
+    result..add(k)..add(v);
```

Comment:
Is this safe, do we now need to check empty string?

Reply:
I don't think this check ever needed to be there - I don't think I added any empty string keys to this map so I am not aware of any new checks that need to be added?

The compiler now requires me to remove this since the `K` type is non-nullable.